File: C_Programming/Day_4/program19.c
//Description : Write a program that accepts number from the user & count its digits

#include<stdio.h>
#include<stdbool.h>

int CountDigits(int iNo)
{
    int iCount = 0;

    while(iNo != 0)
    {
        iCount++;
        iNo = iNo / 10;
    }

    return iCount;

}

int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter the number : ");
    scanf("%d",&iValue);

    iRet = CountDigits(iValue);

    printf("Number of digits are : %d\n",iRet);

    return 0;
}

File: Python_Programming/Day_4/program19.py
# Description : Write a program tha accepts number from the user & count its digits

def CountDigits(No):
    Digit = 0
    Count = 0

    while(No != 0):
        Digit = No % 10
        Count += 1
        No = No // 10

    return Count

def main():

    Value = 0
    Ret = 0

    print("Enter the number : ")
    Value = int(input())

    Ret = CountDigits(Value)

    print("Number of digits are : ",Ret)

if __name__ == "__main__":
    main()